### PR TITLE
Create candu-event-streaming.md

### PR DESCRIPTION
**Pull request by Brandoncharleskhoo1**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -289,6 +289,7 @@ nav:
         - Bento: data/destinations/bento-event-streaming.md
         - Branch: data/destinations/branch.md
         - Braze: data/destinations/braze.md
+        - Candu: data/destinations/candu.md
```
[the remark this answers is not in the document]
Comment (Brandoncharleskhoo1) [reply mkdocs.yml]: Fixed the broken link and renamed it candu-event-streaming.md

File: docs/data/candu-event-streaming.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+2. Enter a sync name, then click **Create Sync**.
+3. Toggle Status from **Disabled** to **Enabled**.
+4. Paste your **API Key (Access Token from the Candu platform)**.
+5. (Optional) In the **Create & Update users** section, enable the toggle if you want to send users and their properties in real-time whenever Amplitude creates a user or updates the user property.
+6. In the **Send Events** section, enable the **Events are sent to Candu** toggle to stream events to Candu. When enabled, Amplitude forwards events to Candu when they're ingested. Events aren't sent on a schedule or on demand using this integration.
+7. In the **Select and filter events** section choose which events you want to send. Choose only the events you need in Candu. This integration doesn't support[Transformed events](https://www.google.com/url?q=https://help.amplitude.com/hc/en-us/articles/5913315221915-Transformations-Retroactively-modify-your-event-data-structure%23:~:text%3DAmplitude%2520Data%27s%2520transformations%2520feature%2520allows,them%2520to%2520all%2520historical%2520data.&sa=D&source=docs&ust=1692341974637179&usg=AOvVaw1BdAYfjzWTy1y9u94STUaQ).
```
Comment (markzegarelli) [inline docs/data/candu-event-streaming.md:40]: Please use the standard URL, not the google highlighted URL. This will break our migration to the new site.

**Review by markzegarelli — APPROVED** [review]

**Review by Brandoncharleskhoo1 — COMMENTED** [review]
UPdated the URL

File: docs/data/candu-event-streaming.md (hunk 2 above)
Comment (Brandoncharleskhoo1) [reply docs/data/candu-event-streaming.md]: Updated the Google highlighted URL.

**Pull request merged**